.github/workflows/test_vendored.yml

```
@@ -5,9 +5,16 @@ on:
   schedule:
     # * is a special character in YAML so you have to quote this string
     - cron:  '0 2 * * *'
```

```suggestion
    - cron:  '0 2 * * 1'    # Mondays, 2AM
```
Shall we save some runner cycles and run this weekly and not daily?